src/auto/flow_box.rs

```diff
@@ -215,7 +215,7 @@ pub trait FlowBoxExt: 'static {
 
 impl<O: IsA<FlowBox>> FlowBoxExt for O {
     //#[cfg(any(feature = "v3_18", feature = "dox"))]
-    //fn bind_model<'a, P: IsA</*Ignored*/gio::ListModel> + 'a, Q: Into<Option<&'a P>>>(&self, model: Q, create_widget_func: /*Unknown conversion*//*Unimplemented*/FlowBoxCreateWidgetFunc, user_data_free_func: /*Unknown conversion*//*Unimplemented*/DestroyNotify) {
+    //fn bind_model<P: Fn(&glib::Object) -> Widget + 'static>(&self, model: /*Ignored*/Option<&gio::ListModel>, create_widget_func: P) {
```

Comment:
Why can't this be generated? Looks like it only need `gio.ListModel` to be added to the manual section of the toml